### DISCUSSION TITLE
0 byte file stuck

### DISCUSSION THREAD

**Discussion by mrcriter** [marcone/teslausb#801] [Q&A]
I'm seeing the file **"2075E8\~2.MP4" (0 bytes) in folder '/TeslaCam/RecentClips/2075E8\~2.M/'**
Is there a way to remove this file or will it cause no problems?
![image](https://github.com/marcone/teslausb/assets/2552971/089e12d6-cff2-44f0-a252-5466502ad7d4)
![image](https://github.com/marcone/teslausb/assets/2552971/e8cfc558-1736-4a74-9123-b9b538a7cdb6)

**Comment by marcone** [comment]
There's no way to remove it, but it won't cause any problems and will be automatically removed at some point.
